### ISSUE TITLE
Operators for Array should not be public

### ISSUE THREAD

**Issue by ShivaHuang** [mattt/Surge#81]
Hi,

I noticed the operator `+` between two`[Float]`s or two `[Double]`s is overridden in `Arithmetic.swift`.

However, I don't think this is a proper design since it changes the default behavior of swift.

For example, if I want to append one array to another, most people will do that in such way:

```swift
let arr: [Double] = [1.0, 2.0]
let newArr: [Double] = arr + [3.0]
```

We expect the `newArr` will be `[1.0, 2.0, 3.0]`, but with `Surge`, it becomes `[4.0]`.

It's hard to debug and to avoid it, we have to write something like this:

```swift
let arr: [Double] = [1.0, 2.0]
var newArr: [Double] = arr
newArr.append(3.0)
```

This forced us to use a variable, not a constant, which is not a good coding habit.

Any suggestions?

**Comment by alejandro-isaza** [comment]
We already have a solution for this on master: https://github.com/mattt/Surge/commit/1e96fab7309d96943bf29415b51b2f24fce46676

Will release a version soon.

**Comment by ShivaHuang** [comment]
Cool! Can't wait for the new version.

**Comment by alejandro-isaza** [comment]
Released 2.1.0
